javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,260 @@
+import "./styles.scss";
+
+// Or if installed from NPM to use with a bundler
+import Artyom from "artyom.js";
+// const artyom = require("artyom.js");
+const artyom = new Artyom();
+
+let catchedName = "";
+const catchedToDo = [];
+const eightBallAnswers = [
+  "It is certain",
+  "It is decidedly so",
+  "Without a doubt",
+  "Yes, definitely",
+  "You may rely on it",
+  "As I see it, yes",
+  "Outlook good",
+  "Yes",
+  "Signs point to yes",
+  "Reply hazy try again",
+  "Ask again later",
+  "Better not tell you now",
+  "Cannot predict now",
+  "Concentrate and ask again",
+  "Don't count on it",
+  "My reply is no",
+  "My sources say no",
+  "Most likely",
+  "Outlook not so good",
+  "Very doubtful",
+];
+
+const getReply = (command) => {
+  const splittedArray = command.split(" ");
+
+  let response = "";
+
+  if (command.toLowerCase().includes("my name is")) {
+    if (catchedName === "") {
+      for (let i = 0; i < splittedArray.length; i++) {
+        if (
+          splittedArray[i] === "my" &&
+          splittedArray[i + 1] === "name" &&
+          splittedArray[i + 2] === "is"
+        ) {
+          catchedName = splittedArray[i + 3];
+          response = `Lovely to meet you ${catchedName}`;
+        }
+      }
+    } else if (catchedName != "") {
```

Comment:
💡 The strict comparison operator is recommend: 
    
`    } else if (catchedName !== "") { `